RubeesEat/Model/DB/DbBillRepository.cs

```diff
@@ -0,0 +1,277 @@
+using System.Collections.Immutable;
+using System.Data;
+
+namespace RubeesEat.Model.DB;
+
+public class DbBillRepository(IDbConnectionFactory connectionFactory) : IBillRepository
+{
+    public IReadOnlyList<Bill> GetAll()
+    {
+        using var connection = connectionFactory.CreateDbConnection();
+        using var command = connection.CreateCommand();
+
+        command.CommandText = "SELECT BillID, Date, Description FROM Bills ORDER BY Date DESC;";
+        command.CommandType = CommandType.Text;
+
+        connection.Open();
+
+        using var reader = command.ExecuteReader();
+
+        var bills = new List<Bill>();
+        while (reader.Read())
+        {
+            var guid = reader.GetGuid(0);
+            bills.Add(new Bill
+            (
+                guid,
+                reader.GetDateTime(1),
+                reader.GetString(2),
+                [..GetEntryLines(guid)]
```

Comment:
This is not good. For each `Bill` object we execute a new command to get all the `EntryLine` instances. If we have 100 `Bill` objects we make 101 requests to the server slowing everything down. Retrieval of `EntryLine` instances should be done in the same request, reducing the total to 1 request.